lib/src/query_component.dart

```diff
@@ -16,9 +17,16 @@ import 'reporting_query_parser.dart';
     <input [(ngModel)]="insertionOrderId" 
            placeholder="Insertion Order ID: 8127549" debugId="io-id-input">
     <br>
+
+    <input type="checkbox" [(ngModel)]="isAdvertiserQuery" name="advertiser-query">
```

Comment:
if the query types are mutually exclusive the affordances should reflect that - typically a radio button